Net SDK Unit Tests/Unit Tests/CommonLibraries/TypeConverterUnitTests.cs

```diff
@@ -77,5 +77,25 @@ public void TruncateToMinutesSuccessfully()
             Assert.AreEqual(0, result.Second);
             Assert.AreEqual(0, result.Millisecond);
         }
+
+        [TestMethod]
```

Comment:
do we need test for this? isn't this part of NET functionality?

Reply:
Nope, this is testing the extension method defined in line 16 of [this file](https://github.com/splitio/.net-client/pull/101/files/4d7fc37ff08da3a43fa84bad4f2d8a4fd6333706#diff-185979129cceed747aea2ab35cef4f13)

Reply:
Good